### ISSUE TITLE
Shading Palette

### ISSUE THREAD

**Issue by Ovicior** [aseprite/aseprite#933]
Pyxel Edit has a shading palette to the right of the color picker. Something like that would be very nice to have in Aseprite.

**Comment by dacap** [comment]
Maybe related to #707

**Comment by Ovicior** [comment]
@dacap Yeah, it's pretty similar. The triangle thing is a really good idea. A Photoshop-like color picker should be lower priority, as Coolorus was meant to improve on it. I don't think the shading idea would be too hard to improve. I would attempt to help code it, but there's a ton of libraries associated with Aseprite and I haven't worked with C++ in a while.

**Comment by rmp135** [comment]
Perhaps this could be a part of the shading tool rather than the colour picker. If you'e picked a single colour Aseprite automatically picks a couple of colours lighter and darker.

**Comment by Cerno-b** [comment]
It would be really helpful if there was a way to bind color changes to certain keys, e.g. increase hue by 3%, that would allow switching to different shades of my current color quickly. May be a bit too complex to implement and too fiddly to configure for many people.
